doc/developers/advanced_installation.rst

```diff
@@ -58,7 +58,7 @@ feature, code or documentation improvement).
    If you plan on submitting a pull-request, you should clone from your fork
    instead.
 
-#. Install a recent version of Python (3.9 or later at the time of writing) for
+#. Install a recent version of Python (|PythonMinVersion| or later) for
```

Comment:
``|PythonMinVersion|`` is not defined because Python min version is not set in `_min_dependencies`. That's why it was hard coded. I don't know if there's a reason against setting python min version in ``_min_dependencies``, maybe we could do it ?

Reply:
OK I naively assumed `PythonMinVersion` would "just work" because it works inside `README.rst` and I thought there was an include somewhere that was doing the magic :sweat_smile: I'll take a closer look ...

Reply:
in README.rst it's defined line 32 ``.. |PythonMinVersion| replace:: 3.10``. We can define it here as well to keep things simple.

Reply:
OK I did it the easier way after looking a bit how this was all generated. Maybe I will do it on a rainy day :sweat_smile: